mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,159 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	you can use this feature in a negated boolean-context fashion:
 	''not (color-gamut)''.
 
+<h3 id="dynamic-range">
+Dynamic Range: the 'dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+High dynamic range (HDR) represents the combination of brightness, color depth, transfer function, and color gamut.
+
+<dl dfn-type=value dfn-for="dynamic-range">
+	<dt><dfn>high</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered high
+		* color depth is greater than 24 bit
+		* has a <a href="#contrast-of-display">high contrast ratio</a>
+	<dt><dfn>standard</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered standards
+		* color depth is 24 bit or 8 bit per color component of RGB
+		* has a <a href="#contrast-of-display">low contrast ratio</a>
+</dl>
+
+<h3 id="contrast-brightness-of-display">
+	Determining contrast and brightness of display</h3>
+
+This text is non-normative. There currently is no agreed upon standardization of
+high contrast ratio, low contrast ratio, max brightness or low brightness as it
+relates to displays. As such, the determination for <a>dynamic-range</a> and
+<a>video-dymanic-range</a> will be defined by the user agent.
+
+<h3 id="video-prefixed-properties">
+Handling bi-plane scenarios</h3>
+
+	In some scenarios, such as TVs, the user agents may have a graphics
```

Comment:
I think the language here could be a little more direct. I couldn't find a great way to do this on a per-word basis, so I'm proposing the following as wholesale replacement. Written in my voice, so some difference in style - please feel free to ignore and/or revise further. Apologies for the large edit.

---

Some user agents, including many TVs, render video and graphics in two separate "planes" (bi-plane) with distinct screen characteristics. A set of video- prefixed features is provided to describe the video plane.

Any bi-plane implementation **MUST** return values based on the video plane for the following features: <a>video-color-gamut</a>; <a>video-width</a>; <a>video-height</a>; <a>video-resolution</a>; <a>video-dynamic-range</a>. All other features **MUST** return values based on the graphics plane. 

Non bi-plane implementations **MUST** return the same values for video-prefixed features and their non-prefixed counterparts.

Reply:
I took your recommendation -- thanks!